cmd/crowdsec-cli/dashboard.go

```diff
@@ -72,7 +67,9 @@ cscli dashboard remove
 				log.Errorf("This command requires direct database access (must be run on the local API machine)")
 				log.Fatalf(err.Error())
 			}
-
+			if err := metabase.TestAvailability(); err != nil {
```

Comment:
Move this check in first.
Absence of docker / arch != amd64 are 'bigger' issues (or at least harder to fix) than having LAPI disabled.